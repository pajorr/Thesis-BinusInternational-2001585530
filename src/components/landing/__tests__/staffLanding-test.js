import React from 'react';
import { shallow } from 'enzyme';

import StaffLanding from '../staffLanding';


describe('userLogin', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<StaffLanding debug/>);

        expect(component).toMatchSnapshot();
    });
});
