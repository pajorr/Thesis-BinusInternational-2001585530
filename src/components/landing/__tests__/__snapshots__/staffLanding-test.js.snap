// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userLogin should render correctly in "debug" mode 1`] = `
<StaffLanding
  classes={
    Object {
      "buttonStyle": "StaffLanding-buttonStyle-1",
      "gridStyle": "StaffLanding-gridStyle-2",
    }
  }
  debug={true}
/>
`;
